### ISSUE TITLE
question about 255 line in prepare_time_data.py

### ISSUE THREAD

**Issue by LingzhengZHANG1** [ChunjingXiao/DiffAD#1]
Nice work! I have a detail question about your code.
when you define the function "insert_normal" in prepare_time_data.py, you use "pre_labels = 'label' ". Why you use "label" instead of "pre_label"?

**Comment by ZehuaGou** [comment]
This occurred because we forgot to make modifications to the test code before uploading. Thank you for bringing this to our attention; we have made the necessary changes and reuploaded it.

**Comment by yanruotian** [comment]
> This occurred because we forgot to make modifications to the test code before uploading. Thank you for bringing this to our attention; we have made the necessary changes and reuploaded it.

Thank you for your work, but that line of code at the moment says 'pre_labels', which I think should be 'pre_label'.

**Comment by ZehuaGou** [comment]
You are correct; it should be 'pre_label'. We have made the necessary changes and re-uploaded it. Thank you very much for pointing out this issue.
